front/app/modules/commercial/project_description_builder/admin/components/Editor/index.tsx

```diff
@@ -39,15 +39,15 @@ const Editor: React.FC<EditorProps> = ({
         Container,
         TwoColumn,
         ThreeColumn,
-        Text,
-        Image,
-        Iframe,
+        TextMultiloc,
+        ImageMultiloc,
+        IframeMultiloc,
         AboutBox,
-        Accordion,
+        AccordionMultiloc,
         WhiteSpace,
         InfoWithAccordions,
         ImageTextCards,
-        Button,
+        ButtonMultiloc,
```

Comment:
Should we eventually rename all of these back to their previous names? E.g. `ButtonMultiloc` -> `Button`?

Reply:
I don't think so, the resolved name is now encoded in the json so renaming those will not be a good idea given we need to migrate the data again to do so. We could only rename the file and keep the resolved name as is but it's descriptive enough as is, so the value would be limited.